src/utils/decrypt.spec.ts

```diff
@@ -1,174 +0,0 @@
-import {
```

Comment:
Would these tests be useful for signer?

Reply:
Most of it is custom stuff that we don't need anymore as we will use more `ethersjs`.